oauth.go

```diff
@@ -31,32 +31,34 @@ func (p *OAuth) Token() (*oauth2.Token, error) {
 	}
 
 	u, _ := url.Parse(EPLogin)
-	q := u.Query()
-	q.Add("expires_in", fmt.Sprint(24*time.Hour/time.Second))
-	q.Add("client_id", cliID)
-	q.Add("grant_type", "password")
-	q.Add("scope", "internal")
-	u.RawQuery = q.Encode()
-
-	v := url.Values{
-		"username": []string{p.Username},
-		"password": []string{p.Password},
+	q := map[string]interface{}{
+		"expires_in":   86400,
+		"client_id":    cliID,
+		"device_token": "34898bf2-3aad-4540-8401-bea572ab8c09",
```

Comment:
@andrewstuart this is the main issue for me. I cannot login with this device token. When I use dev-tools on my different chromes, I get different `device_token`. After using my `device_token`, it works. 

That's probably something that expires on their side, and the one you shared is probably expired. 

I wish there was a way to automatically refresh that in the code. Otherwise my application will keep failing every few days.

I can build a cronjob that fetches it every 12hours or something, and replaces it in the code as an env variable. Thoughts?